proposals/lip-0052.md

```diff
@@ -492,378 +458,817 @@ crossChainTransferParams = {
+
+##### Topics
+
+- `nftID`: the ID of the NFT to be deleted.
+
+##### Data
+
+```java
+deleteNFTEntryEventDataSchema = {
+    "type": "object",
+    "required": ["nftID", "result"],
+    "properties": {
+        "nftID": {
+            "dataType": "bytes",
+            "length": NFT_ID_LENGTH,
+            "fieldNumber": 1
+        },
+        "result": {
+            "dataType": "uint32",
+            "fieldNumber": 2
+        }
+    }
+}
+```
+
+#### createUserEntry
+
+This event has `name = EVENT_NAME_CREATE_USER_ENTRY`, and is emitted when the createUserEntry function is called.
+
+##### Topics
+
+- `address`: the address to be added to the user substore.
+
+##### Data
+
+```java
+createUserEntryEventDataSchema = {
+    "type": "object",
+    "required": ["address", "nftID", "result"],
+    "properties": {
+        "address": {
+            "dataType": "bytes",
+            "length": ADDRESS_LENGTH,
+            "fieldNumber": 1
+        },
+        "nftID": {
+            "dataType": "bytes",
+            "length": NFT_ID_LENGTH,
+            "fieldNumber": 2
+        },
+        "result": {
+            "dataType": "uint32",
+            "fieldNumber": 3
+        }
+    }
+}
+```
+
+#### deleteUserEntry
+
+This event has `name = EVENT_NAME_DELETE_USER_ENTRY`, and is emitted when the deleteUserEntry function is called.
+
+##### Topics
+
+- `address`: the address to be deleted from the user substore.
+
+##### Data
+
+```java
+deleteUserEntryEventDataSchema = {
+    "type": "object",
+    "required": ["address", "nftID", "result"],
+    "properties": {
+        "address": {
+            "dataType": "bytes",
+            "length": ADDRESS_LENGTH,
+            "fieldNumber": 1
+        },
+            "nftID": {
+            "dataType": "bytes",
+            "length": NFT_ID_LENGTH,
+            "fieldNumber": 2
+        },
+        "result": {
+            "dataType": "uint32",
+            "fieldNumber": 3
+        }
+    }
+}
+```
+
+#### create
+
+This event has `name = EVENT_NAME_CREATE`, and is emitted when the create function is called.
+
+##### Topics
+
+##### Data
+
+```java
+createEventDataSchema = {
+    "type": "object",
+    "required": ["address", "collection", "attributes", "result"],
+    "properties": {
+        "address": {
+            "dataType": "bytes",
+            "length": ADDRESS_LENGTH,
+            "fieldNumber": 1
+        },
+        "collection": {
+            "dataType": "uint32",
+            "fieldNumber": 2
+        },
+        "attributes": {
+            "dataType": "bytes",
+            "maxLength": MAX_BYTE_LENGTH_ATTRIBUTES,
+            "fieldNumber": 3
+        },
+        "result": {
+            "dataType": "uint32"
+            "fieldNumber": 4
+        }
+    }
+}
+```
+
+#### destroy
+
+This event has `name = EVENT_NAME_DESTROY`, and is emitted when the destroy function is called.
+
+##### Topics
```

Comment:
Maybe also would be nice to have the address as a topic.

Reply:
Right! Do you also think it should be added to event "data"? For now I only have "nftID" and "result" properties in the case of NFT destruction

Reply:
Okay, I call off my comment about having the address. Probably, we don't need it here for the same reason as above: `nftID` is already given and the destroyer might not have an address (could be some DeFi module).